api/envoy/api/v2/cds.proto

```diff
@@ -417,6 +417,20 @@ message Cluster {
     // Deprecated settings from v1 config.
     // [#not-implemented-hide:] Hide from docs.
     DeprecatedV1 deprecated_v1 = 2 [deprecated = true];
+
+    // The hash function used to hash hosts onto the ketama ring.
+    enum HashFunction {
+      // 'xxHash <https://github.com/Cyan4973/xxHash>', this is the default hash function.
```

Comment:
These docs need link fixing also. Please view the docs here to see how they look.

Reply:
ah, right, let me fix this.